test/python/picongpu/compiling/compileexamples.py

```diff
@@ -0,0 +1,45 @@
+"""
+This file is part of PIConGPU.
+Copyright 2021-2023 PIConGPU contributors
+Authors: Brian Edward Marre
+License: GPLv3+
+"""
+
+from picongpu import pypicongpu
+
+import importlib.util
+import os
+
+import unittest
+
+
+class TestExamples(unittest.TestCase):
+    def load_example_script(self, path):
+        """load and execute example PICMI script from given path"""
+        module_spec = importlib.util.spec_from_file_location("example", path)
+        module = importlib.util.module_from_spec(module_spec)
+        module_spec.loader.exec_module(module)
+
+        sim = module.sim
+
+        return sim
+
+    def build_simulation(self, sim):
+        """build the given instance of simulation"""
+        runner = pypicongpu.Runner(sim)
+        runner.generate(printDirToConsole=True)
+        runner.build()
+
+    def test_LWFA_example(self):
```

Comment:
You could add this as part of the LWFA example PR.

Reply:
Yes, but since it currently is the only test coverage I am hesitant to do that.